sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -5665,6 +6004,61 @@
       "properties": {},
       "lockKey": true,
       "calculateValue": "value = data.offeringIntensity === 'Full Time' ? 'Full-time' : 'Part-time';"
+    },
+    {
+      "input": true,
+      "tableView": false,
+      "key": "isInstitutionBCPublic",
+      "label": "Is institution BC Public",
+      "protected": false,
+      "unique": false,
+      "persistent": false,
+      "type": "hidden",
+      "tags": [],
+      "conditional": {
+        "show": "",
+        "when": null,
+        "eq": ""
+      },
+      "properties": {},
+      "lockKey": true
+    },
+    {
+      "input": true,
+      "tableView": false,
+      "key": "isInstitutionBCPrivate",
+      "label": "Is institution BC Private",
+      "protected": false,
+      "unique": false,
+      "persistent": false,
+      "type": "hidden",
+      "tags": [],
+      "conditional": {
+        "show": "",
+        "when": null,
+        "eq": ""
+      },
+      "properties": {},
+      "lockKey": true
+    },
+    {
+      "input": true,
+      "tableView": false,
+      "key": "allowOnlineDeliveryInputs",

```

Comment:
Was it created with the intention of avoiding executing the check for BC Public/Private in other places? I did not find where it is consumed right now.

Reply:
It was left over, I will remove it.